docker/docker-compose.apps.yml

```diff
@@ -6,34 +6,12 @@ services:
     build:
       dockerfile: docker/proposer.Dockerfile
       context: ..
-    ports:
-      # to use with postman and etc
-      - 8092:8092
+    extra_hosts:
+      - 'host.docker.internal:host-gateway'
     environment:
-      ENVIRONMENT: ${ENVIRONMENT:-localhost}
-      LOG_LEVEL: ${LOG_LEVEL:-debug}
-      LOG_HTTP_PAYLOAD_ENABLED: ${LOG_HTTP_PAYLOAD_ENABLED:-true}
-      LOG_HTTP_FULL_DATA: ${LOG_HTTP_FULL_DATA:-false}
-      BLOCKCHAIN_WS_HOST: ${BLOCKCHAIN_WS_HOST:-blockchain}
-      BLOCKCHAIN_PORT: ${BLOCKCHAIN_PORT:-8546}
-      ENABLE_QUEUE: ${ENABLE_QUEUE:-1}
-      OPTIMIST_HOST: ${OPTIMIST_HOST:-optimist}
-      OPTIMIST_PORT: ${OPTIMIST_PORT:-80}
-      OPTIMIST_WS_PORT: ${OPTIMIST_WS_PORT:-8080}
-      CLIENT_HOST: ${CLIENT_HOST:-client}
-      CLIENT_PORT: ${CLIENT_PORT:-8080}
-    networks:
-      - nightfall_network
+      BLOCKCHAIN_WS_HOST: ${BLOCKCHAIN_WS_HOST:-host.docker.internal}
+      AUTOSTART_RETRIES: ${AUTOSTART_RETRIES:-100}
```

Comment:
I think it makes sense, I'll remove it.

Reply:
Hey @signorecello, I was thinking, it is better to leave this docker-compose file as it is. It makes more sense because we can have the Challenger in it and one can start them whenever one wants.